jenkins/docker-ecr/docker-build-ecr.jenkinsfile

```diff
@@ -0,0 +1,103 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+pipeline {
+    options {
+        timeout(time: 3, unit: 'HOURS')
+    }
+    agent {
+        docker {
+            label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:ubuntu2004-x64-docker-buildx0.6.3-qemu5.0-awscli1.22-jdk14'
+            args '-u root -v /var/run/docker.sock:/var/run/docker.sock'
+        }
+    }
+    parameters {
+        string(
+                defaultValue: 'https://github.com/opensearch-project/opensearch-build',
+                name: 'DOCKER_BUILD_GIT_REPOSITORY',
+                description: 'The git repository name that contains dockerfiles and the docker build script.',
+                trim: true
+        )
+        string(
+                defaultValue: 'main',
+                name: 'DOCKER_BUILD_GIT_REPOSITORY_REFERENCE',
+                description: 'The git reference (branch/tag/commit_id) of above repository.',
+                trim: true
+        )
+        string(
+                defaultValue: 'bash docker/ci/build-image-multi-arch.sh -v <TAG_NAME> -f <DOCKERFILE PATH>',
```

Comment:
I believe the default should be something that if the value is not provided by the user it should run with default value. This is more of an example that should go in the description. Please remove default if you don't have any